pkgs/by-name/ga/gate/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "gate";
+  version = "0.36.7";
+
+  src = fetchFromGitHub {
+    owner = "minekube";
+    repo = "gate";
+    rev = "v${version}";
+    hash = "sha256-WHxpx20O/HuCWqbY4zTxcjyIhW3+FQtTz5sUGAda71g=";
+  };
+
+  vendorHash = "sha256-dswNJQWqN+u/mnpbj9se2j9uEi0ewNTXVlN3WnNbcyg=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
```

Comment:
`with ...` scopes are discouraged. Call lib explicitly, e.g. `lib.licenses` and `lib.maintainers`

Reply:
Should I remove the `with` in `maintainers = with lib.maintainers` too? As in write `lib.maintainers` for each maintainer?

Reply:
you can usse `with lib.maintainers;`, that seems to be accepted